src/containers/Application/ElementsBox.tsx

```diff
@@ -49,6 +50,7 @@ const ElementsBox: React.FC<ElementsBoxProps> = ({
           isEditable={elementsState[question.code].isEditable}
           isRequired={elementsState[question.code].isRequired}
           allResponses={responsesByCode}
+          evaluator={evaluateExpression}
```

Comment:
I think a better place for import of this one is in ApplicationViewWrapper (not a required change though)

Reply:
Done.